### PR TITLE
Tpetra: Fixing WrappedDualView usage bug

### PR DESCRIPTION
**Pull request by csiefer2**
Issue: #9937
Tests: Fixing it
Stakeholder feedback: This works now

### CODE REVIEW

File: packages/tpetra/core/example/Finite-Element-Assembly/fem_assembly_InsertGlobalIndices_FE.hpp [hunk 1]
```diff
@@ -131,6 +131,8 @@ int executeInsertGlobalIndicesFESP_(const Teuchos::RCP<const Teuchos::Comm<int>
   int nex = opts.numElementsX;
   int ney = opts.numElementsY;
 
+  printf("CMS: nex = %d ney = %d procx = %d procy = %d\n",nex,ney,procx,procy);
```
Comment (brian-kelley) [inline packages/tpetra/core/example/Finite-Element-Assembly/fem_assembly_InsertGlobalIndices_FE.hpp:134]: Remove debug printout

**Review by brian-kelley — CHANGES_REQUESTED** [review]
Thanks for fixing! Just see a temporary debug printout that was committed

File: packages/tpetra/core/example/Finite-Element-Assembly/fem_assembly_InsertGlobalIndices_FE.hpp (hunk 1 above)
Comment (csiefer2) [reply packages/tpetra/core/example/Finite-Element-Assembly/fem_assembly_InsertGlobalIndices_FE.hpp]: Fixed

**Review by brian-kelley — APPROVED** [review]

**Review by brian-kelley — APPROVED** [review]

**Comment by william76** [comment]
@csiefer2 @kddevin @brian-kelley 
The commit: https://github.com/trilinos/Trilinos/pull/9938/commits/13f43f5624bedcebf63a15a0089aa21f475abf29 
is causing problems for the autotester since it's coming from an unknown user and it's tripping some security protocols. 

@trilinos/framework recommends that the best course of action here is to re-form your branch without an _unknown_ commit author and re-submit the PR.

**Comment by william76** [comment]
Closing this PR out.

@kddevin @csiefer2 @brian-kelley @jwillenbring @e10harvey : see the email I sent you guys for more information.

**Pull request closed without merging**